README.md

```diff
@@ -6,13 +6,20 @@ You can follow the tutorial to learn how to integrate Authgear with your React s
 
 ## Environment setup
 
-Setup the `endpoint` and `clientID` variables in `src/index.tsx`.
+Setup the `endpoint` and `clientID` variables in a `.env` file using the following fields:
+```
+AUTHGEAR_CLIENT_ID=your Authgear app client id
+AUTHGEAR_ENDPOINT=your Authgear project URL
+AUTHGEAR_REDIRECT_URL=your redirect URI
```

Comment:
Must prefixed with `VITE_`

Reply:
mention the redirect url is `http://localhost:4000/auth-redirect` for this project?